.semaphore/semaphore.yml

```diff
@@ -4,20 +4,25 @@ agent:
   machine:
     type: e1-standard-2
     os_image: ubuntu1804
-
 blocks:
-  - name: "🎩 Yarn"
+  - name: "\U0001F3A9 Yarn"
```

Comment:
huh, does this actually make a difference?

Reply:
Uh... weird.  Not sure what made that change, but it was applied automatically by something...

Reply:
prettier most likely

Reply:
Aha, it was Semaphore! It makes the commit when you grant access to the secret.